jobs/sync-stream-metadata.Jenkinsfile

```diff
@@ -8,13 +8,28 @@ properties([
     ])
 ])
 
-cosaPod {
+cosaPod(configMaps: ["fedora-messaging-cfg"], secrets: ["fedora-messaging-coreos-key"]) {
     git(url: 'https://github.com/coreos/fedora-coreos-streams', credentialsId: 'github-coreosbot-token')
     withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', credentialsId: 'aws-fcos-builds-bot']]) {
-        shwrap("""
+        // XXX: update configMap to use the /run/secrets value
+        shwrap("ln -sf \${FEDORA_MESSAGING_COREOS_KEY} /run/fedora-messaging-coreos-key")
+
+        // XXX: eventually we want this as part of the pod or built into the image we use
+        shwrap("git clone --depth=1 https://github.com/coreos/fedora-coreos-releng-automation /var/tmp/fcos-releng")
+
+        cmd = """
             aws s3 sync --acl public-read --cache-control 'max-age=60' \
                 --exclude '*' --include 'streams/*' --include 'updates/*' \
                     ./ s3://fcos-builds
-        """)
+        """
+
+        def dry_run = shwrapCapture("${cmd} --dryrun").trim()
+        if (dry_run != "") {
```

Comment:
so if you run it with `--dry-run` and there's output then we need to send a message? Could we just capture the output of the command without dry-run and have it work? either way might be worth a comment.

Reply:
I think that would work, but I don't like the idea of capturing output of the real run (though I guess we could print it out, but I find this cleaner). Added a comment!

Reply:
👍